pkg/cmd/login/html.go

```diff
@@ -98,7 +98,100 @@ var loginHTML = []byte(`
         </g>
       </svg>
       <h1>You are now logged in!</h1>
-      <h2>Your session is now active in the Okteto CLI. You may close this window.</h2>
+      <h2>Your session is now active in the Okteto CLI</h2>
+      <h2>You may close this window</h2>
+    </div>
+  </body>
+</html>
+`)
+
+var loginHTMLDockerDesktop = []byte(`
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="refresh" content="1; url = docker-desktop://dashboard/open">
```

Comment:
This is what implements the redirect to docker desktop after 1s

Reply:
Instead of duplicating the template, you can add a templated string and pass the full " <meta http-equiv="refresh" content="1; url = docker-desktop://dashboard/open">" as an optional value.

Reply:
yes!